js/engine/tests/__snapshots__/index.test.js.snap

```diff
@@ -22,26 +22,24 @@ exports[`yaml parser parses a pattern with pattern-regex 1`] = `
       },
       "matches": [
         {
+          "check_id": "test",
```

Comment:
FYI @tpetr some snapshot update because of changes in semgrep_output_v1.atd (the semgrep-core output spec part of it)